lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -216,6 +216,28 @@ class ConfigSchemaHandler {
     });
   }
 
+  extendFunctionEventDefinition(providerName, name, configSchema) {
```

Comment:
To match convention of other methods, let's name it `defineFunctionEventProperties`